src/Http/Controllers/PaymentCallbackController.php.stub

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Solutionplus\Payable\Http\Controllers;
+
+use Solutionplus\Payable\Contacts\PaymentCallbackContract;
+
+class PaymentCallbackController extends Controller implements PaymentCallbackContract
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function handleCallback(): bool
+    {
+        return true;
```

Comment:
**Note**

we may return empty response here instead of boolean 

in payment project we will check just for status code